src/approaches/interactive_learning_approach.py

```diff
@@ -227,6 +227,9 @@ def _create_glib_interaction_strategy(
         self, train_task_idx: int
     ) -> Tuple[Callable[[State], Action], Callable[[State], bool]]:
         """Find the most interesting reachable ground goal and plan to it."""
+        # TODO: when refactoring, give any goals that have only already-known
```

Comment:
wait why? the trajectory that gets you to that goal could still be very useful from a score function perspective

Reply:
that's true... anyway, that would be a functional change

Reply:
okay i'll make this functional change and we'll see if there is a noticeable difference in the supercloud

Reply:
actually nvm, this breaks some things, so I'm gonna try to stay true to the original implementation